addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -270,7 +270,7 @@ export class MassMailingHtmlField extends HtmlField {
             });
             this.mobilePreview = this.dialog.add(MassMailingMobilePreviewDialog, {
                 title: _t("Mobile Preview"),
-                preview: mailingHtml.body.innerHTML,
+                preview: markup(mailingHtml.body.innerHTML),
```

Comment:
I don't know why it doesn't appears in the preview commit 7f5a0ccf9e5fdf3f7fece8a9f255ac6415d9dc79 but we would like OFC to markup props.preview.